composer.json

```diff
@@ -3,6 +3,9 @@
     "license": "MIT",
     "type": "project",
     "description": "An app for storing data about servers",
+    "optimize-autoloader": true,
+    "classmap-authoritative": true,
+    "apcu-autoloader": true,
```

Comment:
https://getcomposer.org/doc/articles/autoloader-optimization.md
as i understand we can use any of these methods